package.json

```diff
@@ -9,7 +9,8 @@
     "jest-expo": "25.0.0",
     "prettier": "git+https://github.com/Automattic/calypso-prettier.git#calypso-1.9",
     "react-native-scripts": "1.11.1",
-    "react-test-renderer": "16.2.0"
+    "react-test-renderer": "16.2.0",
+    "remote-redux-devtools": "^0.5.12"
```

Comment:
Not sure how this works but, if it's a dev only dep, will importing it into `store/index.js` (as below) become broken in production?

Reply:
Good point, I'll have to add a check and enable it for dev mode only.

Reply:
https://github.com/zalmoxisus/redux-devtools-extension#14-using-in-production might be relevant.

Reply:
Testing, will delete.  Please ignore.

Reply:
Updated in 8692a96

Reply:
Verified it doesn't load in production mode.